pkg/autoops/domain/auto_ops_rule.go

```diff
@@ -82,11 +117,21 @@ func (a *AutoOpsRule) SetDeleted() {
 func (a *AutoOpsRule) SetTriggeredAt() {
 	now := time.Now().Unix()
 	a.AutoOpsRule.TriggeredAt = now
+	a.AutoOpsStatus = proto.AutoOpsStatus_COMPLETED
 	a.AutoOpsRule.UpdatedAt = now
 }
 
+func (a *AutoOpsRule) SetCompleted() {
```

Comment:
Can you rename the proto from `COMPLETED` to `FINISHED`?
The tab on the console is using `Finished`, and also the Progressive Rollout is using `FINISHED`.

[Reference](https://github.com/bucketeer-io/bucketeer/blob/main/proto/autoops/progressive_rollout.proto#L30).